src/wallet/rpcwallet.cpp

```diff
@@ -2716,9 +2716,18 @@ UniValue mint(const UniValue& params, bool fHelp)
 
     if (fHelp || params.size() != 1)
         throw std::runtime_error(
-            "mint <amount>(could not smaller than 0.1)\n"
-            "\nAutomatically choose denominations to mint by amount\n" +
-            HelpRequiringPassphrase());
+            "mint amount\n"
+            "\nAutomatically choose denominations to mint by amount."
+            + HelpRequiringPassphrase() + "\n"
+            "\nArguments:\n"
+            "1. \"amount\"      (numeric or string, required) The amount in " + CURRENCY_UNIT + " to mint but must not less than 0.1\n"
```

Comment:
I think `must not less than 0.1` is not correct. It must be something like `must be able to multiply with 0.1`.